tests/cloudlaunchprocess/test.csv

```diff
@@ -0,0 +1,25 @@
+action,ms,bytes,lang,code,repeat,minver,comment
+comment,0,,"Test 1 - Launch and then kill a process with all possible cmd line args"
+run,0,0,q,.sys.launch input1,,,"Start a new process with extra command line args"
+run,0,0,q,system "sleep 2",,,"Wait for process to start"
+run,0,0,q,h:hopen `::7124:admin:admin,,,"Attempt to connect to the new process"
+true,0,0,q,`test2=@[h;".proc.procname";`],,,"Check that connection has been made"
+true,0,0,q,11011111010b~(value input1) in' value key[input1]#.Q.opt (h)".z.X",,,"Tentatively? - Check that the cmd line args are as expected"
```

Comment:
```suggestion
true,0,0,q,11011111010b~(value input1) in' value key[input1]#.Q.opt h".z.X",,,"Tentatively? - Check that the cmd line args are as expected"
```

don't need those parentheses

also the comment says "Tentatively?" - does that need to be updated?